src/functions/template-tags/general.php

```diff
@@ -1220,6 +1220,95 @@ function tribe_events_event_schedule_details( $event = null, $before = '', $afte
 		return apply_filters( 'tribe_events_event_schedule_details', $schedule, $event->ID, $before, $after );
 	}
 
+	/**
+	 * Return the short details of the start/end date/time.
+	 *
+	 * @see tribe_events_event_schedule_details() for the format of the schedule details.
+	 *
+	 * @param int|null $event  The event post ID, or `null` to use the global event.
+	 * @param string   $before A string to prepend before the schedule details.
+	 * @param string   $after  A string to append after the schedule details.
+	 * @param bool     $html   Whether to use HTML elements in the output string or not; defaults to `true`.
```

Comment:
I might add a note that this only applies to non-multiday and non-all-day events?